libtransmission/torrent.c

```diff
@@ -1131,6 +1132,72 @@ static tr_parse_result torrentParseImpl(tr_ctor const* ctor, tr_info* setmeInfo,
     return result;
 }
 
+static void tr_torrentAddDefaultTrackers(tr_torrent* tor)
+{
+    tr_list* l;
+    int tier = -1;
+    int i;
+    tr_tracker_info* trackers;
+    int numExistingTrackers = tor->info.trackerCount;
+    int numNewTrackers = tr_list_size(tor->session->defaultTrackers);
+    char* url;
+    bool duplicate;
+    bool changed = false;
+
+    if (!numNewTrackers || tor->info.isPrivate)
```

Comment:
it's slightly cleaner to use the `tr_torrentIsPrivate(tor)` helper for the second test

Reply:
Changed.